dns/dnscrypt-proxy/src/opnsense/mvc/app/controllers/OPNsense/Dnscryptproxy/forms/general.xml

```diff
@@ -179,4 +179,12 @@
         <allownew>true</allownew>
         <help><![CDATA[Exclude servers from automatic selection. Add any specific server names here if you do not want to use them for any reason.]]></help>
     </field>
+    <field>
+        <id>general.relaylist</id>
+        <label>Relay List</label>
+        <type>select_multiple</type>
+        <style>tokenize</style>
+        <allownew>true</allownew>
+        <help><![CDATA[Set a list of relays. Will be used for relaying to all configured servers.]]></help>
```

Comment:
@mimugmail The help information which you are requesting is already available, or am I missing something?

Reply:
Ah sorry, did not read your comment fully, will add the link to the list of available relays.

Reply:
Okay, made the change you've requested. Be aware, the I pushed a fixup commit, so probably a squash merge will be required. If I should squash locally and repush instead, then please let me know.

Reply:
I didn't understand how to add the server relays for anonymous DNS.
Add the url link or the server?

Reply:
No, you insert the name of the relay which you can get from the URL

Reply:
> 

Ok! Uncheck option Servers DNS-over-HTTPS?
Insert one or multiple relay?

Reply:
Seems @schreibubi needs to write a doc PR too :)